protocol/messenger_communities.go

```diff
@@ -1238,15 +1239,6 @@ func (m *Messenger) CreateCommunity(request *requests.CreateCommunity, createDef
 		response.AddChat(chatResponse.Chats()[0])
 	}
 
-	if request.Encrypted {
-		// Init hash ratchet for community
-		_, err = m.encryptor.GenerateHashRatchetKey(community.ID())
-
-		if err != nil {
-			return nil, err
-		}
-	}
```

Comment:
Just want to make sure: this means, we can never create a community and have it encrypted right away. It will always be unencrypted by default.

This is desired, yes?